helpers/email/email.py

```diff
@@ -14,7 +14,7 @@ def send_email_notification(
         """
         send email notifications after a given activity has occured
         """
-        recipients = [kwargs.get('email')]
+        recipients = kwargs.get('email')
```

Comment:
@mwinel The other functions that use the `send_email_notification` method rely on the fact that `kwargs.get('email')`  returns a string and not a list of strings. Changing it here would affect those functions. Kindly find a way around this situation.